### ISSUE TITLE
Test failure: Interop\\IJW\\NativeVarargs\\NativeVarargsTest\\NativeVarargsTest.cmd

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#36966]
failed in job: [runtime-coreclr jitstress2-jitstressregs 20200524.1](https://dev.azure.com/dnceng/public/_build/results?buildId=658473&view=ms.vss-test-web.build-test-results-tab&runId=20435328&resultId=100380&paneView=debug)

Error message
~~~
Return code: 1
Raw output file: C:\h\w\A6980933\w\C0510A35\e\Interop\IJW\Reports\Interop.IJW\NativeVarargs\NativeVarargsTest\NativeVarargsTest.output.txt
Raw output:
BEGIN EXECUTION
 "C:\h\w\A6980933\p\corerun.exe" NativeVarargsTest.dll 
Failed Varargs tests:
SumSumInt64s
RunSumInt64sTest Failed:Expected: 100
Actual: 102
END EXECUTION - FAILED
FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=C:\h\w\A6980933\p
> C:\h\w\A6980933\w\C0510A35\e\Interop\IJW\NativeVarargs\NativeVarargsTest\NativeVarargsTest.cmd
Expected: True
Actual: False


Stack trace
   at Interop_IJW._NativeVarargs_NativeVarargsTest_NativeVarargsTest_._NativeVarargs_NativeVarargsTest_NativeVarargsTest_cmd() in F:\workspace\_work\1\s\artifacts\tests\coreclr\Windows_NT.x86.Checked\TestWrappers\Interop.IJW\Interop.IJW.XUnitWrapper.cs:line 625
~~~

**Comment by BruceForstall** [comment]
Failure mode: `CoreCLR Windows_NT x86 Checked jitstress2_jitstressregs4 @ Windows.10.Amd64.Open`

@dotnet/jit-contrib

**Comment by CarolEidt** [comment]
Fixed by #36994 or #37062
